libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -231,6 +233,86 @@ describe('ModalWrapperComponent', () => {
     });
   });
 
+  describe('with embedded page progress component', () => {
+    describe('with static page progress', () => {
+      beforeEach(() => {
+        spectator = modalWrapperTestBuilder
+          .flavor('modal')
+          .withStaticPageProgress()
+          .build();
+        spectator.detectComponentChanges();
+      });
+
+      afterEach(() => {
+        // Ensure any observers are destroyed:
+        spectator.fixture.destroy();
+      });
+
+      it('should move embedded page progress content to ion toolbar', () => {
+        const ionContentElement = spectator.query('ion-content');
+        const embeddedComponentElement = ionContentElement.firstElementChild;
+
+        const embeddedPageProgress = embeddedComponentElement.querySelector('kirby-page-progress');
+        expect(embeddedPageProgress).toBeNull();
+
+        const pageProgressAsWrapperChild = spectator.element.querySelector('kirby-page-progress');
+        expect(pageProgressAsWrapperChild).not.toBeNull();
```

Comment:
Please group in Arrange-Act-Assert
```suggestion
        const embeddedComponentElement = ionContentElement.firstElementChild;
        const embeddedPageProgress = embeddedComponentElement.querySelector('kirby-page-progress');
        const pageProgressAsWrapperChild = spectator.element.querySelector('kirby-page-progress');
        
        expect(embeddedPageProgress).toBeNull();
        expect(pageProgressAsWrapperChild).not.toBeNull();
```